gradle/libs.gradle

```diff
@@ -1,4 +1,5 @@
 project.ext.libs = [
+    'bcpkix': 'org.bouncycastle:bcpkix-jdk15on:1.54',
```

Comment:
Figured out how to do compile-only dependency from Gradle article posted yesterday (http://gradle.org/compile-only-dependencies/) -- this was just introduced in Gradle 2.12, which was released on March 14. Hooray for timeliness!

I dug around on this a bit more, and figured out another way to do this that doesn't require BouncyCastle. Unfortunately, it uses `sun.security.util.*` classes, which are also not completely portable (but more so than BouncyCastle?). I may add another commit that falls back on the Sun behavior if BouncyCastle isn't present, but that requires some Checkstyle suppressions to allow the Sun import.

So annoying that there isn't a lighter-weight library for just parsing DER/ASN.1 streams... I may just implement a simple one myself to try to work around this, but that would be more of a personal project, so this approach will have to do for now :)